cdc/sorter/leveldb/db/db.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package db
```

Comment:
Better to move the interface definition to separated path, instead of the sub-path of leveldb

Reply:
Besides do we need a concrete package name, `db` is confused.

Reply:
Any suggestion about the name?

Reply:
I'm going to move these files to pkg/db, and name it package db. Feel free to suggest a better name, thanks!